ts/src/core/parser/index.ts

```
@@ -50,89 +143,318 @@ async function parseJSONLFile(
+      // Convert to local time
+      const localEventTime = new Date(eventTime.toLocaleString());
+      
+      if (localEventTime >= startTime && localEventTime <= endTime) {
+        events.push({
+          ...event,
+          timestamp: eventTime.toISOString(),
+        });
       }
     } catch (error) {
       // Skip invalid lines
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)